internal/host/transition_test.go

```diff
@@ -1663,7 +1691,7 @@ var _ = Describe("Refresh Host", func() {
 					HasMinMemory:                {status: ValidationSuccess, messagePattern: "Sufficient minimum RAM"},
 					HasMinValidDisks:            {status: ValidationSuccess, messagePattern: "Sufficient disk capacity"},
 					IsMachineCidrDefined:        {status: ValidationSuccess, messagePattern: "Machine Network CIDR is defined"},
-					HasMemoryForRole:            {status: ValidationFailure, messagePattern: "Require at least 16 GiB RAM role master, found only 16 GiB"},
+					HasMemoryForRole:            {status: ValidationSuccess, messagePattern: "Sufficient RAM for role master"},
```

Comment:
What has changed in this test case that now this validation is successful?

Reply:
There is exactly 16GiB of memory available and required, so it should be a success and as the purpose of that test was check virtualization capabilities IMO it's OK not to mock the original behavior.